features/logging/logging_acceptance.feature

```diff
@@ -0,0 +1,163 @@
+@clusterlogging
+@commonlogging
+Feature: Logging smoke test case
+
+  # @author gkarager@redhat.com
+  # @case_id OCP-37508
+  @admin
+  @destructive
+  Scenario: One logging acceptance case for all cluster
+# Deploy cluster-logging operator via web console
+    Given logging service is removed successfully	
+    Given the logging operators are redeployed after scenario	
```

Comment:
I think we don't need this step if the case won't be executed with other logging cases. @anpingli could you please confirm?

Reply:
@QiaolingTang  @gkarager  Let's keep this ' Given logging service is removed successfully' before I disable the logging operator deployment in flex.   
I think we can drop this step 'Given the logging operators are redeployed after scenario'